docs/guide/security_checklist.md

```diff
@@ -4,65 +4,192 @@ description: Ensure that your Ibexa DXP installation is secure by following our
+
+## [[= product_name =]]
+
+### Carefully select admin users
+
+Make sure Admin users and other privileged users who have access to System Information and setup in the back end are vetted and fully trustworthy.
+
+As administrator you have access to full information about the system through the `setup/system_info` policy, and also to user data, role editing, and many other critical aspects.
+
+The users in your organization who have backend access must be kept up-to-date.
+Any user leaving the organization must be disabled without delay.
+If a user takes on a new role in the organization, any required role changes for them in [[= product_name =]] must also be made as soon as possible.
+
+### Strong passwords
+
+Enforce strong passwords for all users.
+This is specially important for admin accounts and other privileged users.
+
+- Never go online with admin password set to `publish` or any other default value.
+- Introduce password quality checks. Make sure the checks are strict enough (length/complexity).
+- 16 characters is a quite secure minimum length. Don't go below 10.
+- If using [[= product_name =]] v4.5 or newer, enable the password rule that rejects any password which has been exposed in a public breach.
+
+!!! tip "Password rules"
+
+    See [setting up password rules](passwords.md#password-rules).
+
+### Protect against brute force attacks
+
+Consider introducing a measure against brute force login attacks, like CAPTCHA.
+Adjust timeout limits to your needs:
+
+When using the "forgot password" feature, a token is created which expires if the user doesn't click the password reset link that gets mailed to them in time.
+The time before it expires is set in the parameter `ibexa.site_access.config.default.security.token_interval_spec`.
+By nature this feature must be available to users before they have logged in, including would-be attackers.
+If an attacker uses this feature with someone else's email address, the attacker doesn't receive the email.
+But they could still try to guess the password reset link.
+That's why this interval should be as short as possible.
+5 minutes is often enough.
+
+[[= product_name =]] allows you to create and send invitations to create an account in the frontend as a customer, the back office
+as an employee, or the Corporate Portal as a business partner.
+You can send invitations to individual users or in bulk.
+These invitations time out according to the parameter
+`ibexa.site_access.config.default.user_invitation.hash_expiration_time`.
+This can safely be longer than the "forgot password" time, since attackers cannot generate invitations.
+Don't leave it longer than it needs to be, though.
+
+These timeouts are both entered as [PHP DateInterval duration strings](https://www.php.net/manual/en/dateinterval.construct.php).
+The forgot password feature defaults to "PT1H" (one hour).
+The account invitation feature defaults to "P7D" (seven days).
```

Comment:
This feature seems added circa 4.2 according to
https://github.com/ibexa/user/pull/35/files#diff-d7dd8ed9e114fd1e91bb9a9c4ee717a918265fa26cd2d152dbd14a909408fd59R40 and its ticket.

The remaining part may need rewording.

```suggestion
This timeout is entered as [PHP DateInterval duration strings](https://www.php.net/manual/en/dateinterval.construct.php).
The forgot password feature defaults to "PT1H" (one hour).
```

Reply:
Nice catch. Edit looks good. Maybe "as a [PHP DateInterval duration string]"